src/components/MainContent.tsx

```diff
@@ -99,6 +99,7 @@ const MainContent = ({
               width: '100%',
               maxWidth: hasFrame ? 1140 : 'initial',
               flexWrap: 'nowrap',
+              overflow: 'hidden',
```

Comment:
This seems to break the focus ring from showing in the side nav correctly; being "clipped" off.
<img width="430" alt="image" src="https://user-images.githubusercontent.com/53757053/141424448-d58d6a41-6161-4c62-9799-38f0b0001fe3.png">

Reply:
This should be fixed now with ca07641d9691ae541d8c8e348c50c9e0160cb933